sdk/confidentialledger/azure-security-confidentialledger/README.md

```diff
@@ -21,7 +21,7 @@ portfolio, Azure Confidential Ledger runs in SGX enclaves. It is built on Micros
 <dependency>
   <groupId>com.azure</groupId>
   <artifactId>azure-security-confidentialledger</artifactId>
-  <version>1.0.0-beta.2</version>
+  <version>1.0.0</version>
```

Comment:
So instead of `com.azure:azure-security-confidentialledger;1.0.0-beta.2;1.0.0`, the `version_client.txt` should say `com.azure:azure-security-confidentialledger;1.0.0;1.0.0`?

Reply:
What you currently have in version_client.txt is correct!